resources/js/components/shared/ModalSaveVersion.vue

```diff
@@ -306,10 +307,60 @@ export default {
     });
     this.retrieveSavedSearchCharts();
     this.getProcessDescription();
+
     //Receives selected Option from launchpad Icons multiselect
     this.$root.$on("launchpadIcon", this.launchpadIconSelected);
   },
   methods: {
+    /**
+     * Get all information related to Launchpad Settings Modal
+     */
+    getLaunchpadSettings() {
+      this.images = [];
+      ProcessMaker.apiClient
+        .get("processes/" + this.processId + "/media")
+        .then((response) => {
+          const launchpadProperties = JSON.parse(
+            response.data.data[0]?.launchpad_properties
```

Comment:
0 is hardcode

Reply:
It's hardcoded but, response always will have one element in the array "response.data.data" because API search by Process ID.